src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -70,7 +70,7 @@ public Ticket createTicket(@RequestBody Ticket ticket){
         try {
             newTicket= ticketService.save(ticket);
         } catch (Exception e) {
-
+            e.printStackTrace();
```

Comment:
Handling exceptions well is a good idea.

Use proper logging instead of printStackTrace().
- Loggins has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.